filesystem.go

```diff
@@ -152,11 +152,6 @@ func (fs *filesystem) createAllResources(pod Pod) error {
 }
 
 func (fs *filesystem) storeFile(path string, data interface{}) error {
-	_, err := os.Stat(path)
```

Comment:
BTW, @sameo - I don't really understand that `os.Stat(podlockFile)` call above - again, it seems like it *could* be dropped, or could it...? It looks racy though.